### ISSUE TITLE
Socker error

### ISSUE THREAD

**Issue by huseyinkozan** [huseyinkozan/tessel-network-example#1]
After a `t2 push` and manually start the app:
```
$ ssh -i ~/.tessel/id_rsa root@192.168.2.101


BusyBox v1.23.2 (2017-05-24 04:56:54 +03) built-in shell (ash)

Tessel 2  /  Built on OpenWrt
root@t:~# pgrep node
root@t:~# /app/start 
Hw();
setDBPath(/app/remote-script/database);
loadData(configuration,configuration.json)
setupWireless(user_command); user_command: false
setupWifi();
disableAP();
setupButtonsAndLeds();
Listening port 80
enableWifi(user_command); user_command:false
touchDBFile(configuration.json)
setupWifiDisconnect();
syncTime();
GET / 200 221.251 ms - 8247
GET /assets/jquery/dist/jquery.min.js 304 122.771 ms - -
GET /index.css 304 85.815 ms - -
GET /assets/toastr/toastr.css 304 110.846 ms - -
GET /assets/semantic-nogooglefonts/dist/semantic.min.css 304 109.245 ms - -
GET /common.js 304 10.784 ms - -
GET /assets/toastr/toastr.js 304 35.735 ms - -
GET /assets/semantic-nogooglefonts/dist/semantic.min.js 304 36.295 ms - -
GET /index.js 304 19.588 ms - -
GET /assets/semantic-nogooglefonts/dist/themes/default/assets/fonts/icons.woff2 304 13.285 ms - -
is_wifi_connected();
last_wifi_error();
GET /api/configuration 200 27.551 ms - 167
Socket: Error occurred,  { Error: read ECONNRESET
    at exports._errnoException (util.js:1018:11)
    at Pipe.onread (net.js:568:26) code: 'ECONNRESET', errno: 'ECONNRESET', syscall: 'read' }
Socket: Error occurred,  { Error: read ECONNRESET
    at exports._errnoException (util.js:1018:11)
    at Pipe.onread (net.js:568:26) code: 'ECONNRESET', errno: 'ECONNRESET', syscall: 'read' }
/usr/lib/node/tessel-export.js:255
      throw new Error('Socket: The Port socket has closed.');
      ^

Error: Socket: The Port socket has closed.
    at Socket.sock.on (/usr/lib/node/tessel-export.js:255:13)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:497:12)
root@t:~# 
```

**Comment by huseyinkozan** [comment]
Tessel image is node-v6.10.3:
https://github.com/tessel/openwrt-tessel/pull/73
Build at : https://github.com/tessel/openwrt-tessel/pull/73/commits/59ca9c97a275e6a970c1ee668a5591514ab1cd74

**Comment by huseyinkozan** [comment]
It seems this is related with 6.10 image. It works with 6.5.

**Comment by huseyinkozan** [comment]
Maybe related to this:
https://github.com/tessel/t2-firmware/commit/fe9165c7137a8c0fe5809c282e39f8f267ec92e1

Which is fixed at t2-firmware 0.0.16, but image is at 0.0.15.
